server/src/main/java/org/opensearch/http/HttpChunk.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.http;
+
+import org.opensearch.common.annotation.PublicApi;
+import org.opensearch.common.lease.Releasable;
+import org.opensearch.core.common.bytes.BytesReference;
+
+/**
+ * Represents a chunk of the HTTP request / response stream
+ */
+@PublicApi(since = "2.15.0")
```

Comment:
Can we start with `@ExperimentalApi` since I suspect a lot of this might still be evolving? Same applies to other annotated classes here.

Reply:
Yes we can, sure